modules/network/src/testFixtures/java/org/apache/ignite/internal/network/utils/ClusterServiceTestUtils.java

```diff
@@ -178,24 +179,26 @@ public CompletableFuture<Void> start() {
                                 )
                 ).join();
 
-                bootstrapFactory.start();
+                bootstrapFactory.startAsync();
 
-                clusterSvc.start();
+                clusterSvc.startAsync();
 
                 return nullCompletedFuture();
             }
 
             @Override
-            public void stop() {
+            public CompletableFuture<Void> stopAsync() {
                 try {
                     IgniteUtils.closeAll(
```

Comment:
How are we going to handle async exceptions inside closeAll? I mean that it was designed to close as much as possible, semi-ignoring pending exceptions.